pkgs/by-name/ni/nixedit/package.nix

```diff
@@ -0,0 +1,73 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  bash,
+  fzf,
+  jq,
+  micro,
+  git,
+  nix-tree,
+  coreutils,
+  makeWrapper,
+  dialog,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "nixedit";
+  version = "1.0.0";
```

Comment:
The upstream repo has no tags. How do you know  this is 1.0.0?
("It is in my mind, I am an upstream developer"  is not acceptable.)